tests/csm/rest/test_csm_users.py

```diff
@@ -406,22 +419,24 @@ def test_5001(self):
         resp_error_code = test_cfg["error_code"]
         resp_msg_id = test_cfg["message_id"]
         resp_data = self.rest_resp_conf[resp_error_code][resp_msg_id]
-        msg = resp_data[0]
+        resp_msg_index = test_cfg["message_index"]
+        msg = resp_data[resp_msg_index]
         response = self.csm_obj.list_csm_users(
             expect_status_code=const.BAD_REQUEST,
             sort_by=invalid_sortby,
-            verify_negative_scenario=True)
-        self.log.info("Response : %s", response)
+            return_actual_response=True)
         self.log.info("Verifying the response for invalid value for sort_by")
-        assert response, "Status code check has failed check has failed."
-        self.log.info("Verified the response for invalid value for sort_by")
-
+        assert_utils.assert_equals(response.status_code, const.BAD_REQUEST)
```

Comment:
Can this be HTTPStatus.BAD_REQUEST?

Reply:
will have a ticket to remove where we used commons\constants under our rest api tests